### PR TITLE
Show motd on ssh login

### PR DESCRIPTION
**Pull request by scoopex**
Currently the motd is not displayed when performing a SSH login.

As logins in CSP environments are primarily carried out via SSH, we activate this here, as otherwise the configuration of a Motd would make little sense. This change ignores Redhat environments, because i do not have access to such a environment.

### CODE REVIEW

**Comment by berendt** [comment]
Make this configurable and disable it by default. We had this in the past and a lot of customers didn't like it.

File: roles/motd/tasks/configure-Debian-family.yml [hunk 1]
```diff
@@ -38,9 +38,19 @@
   loop: "{{ etc_pam_motd.files }}"
 
 - name: Configure SSH to print the motd
+  become: true
   copy:
     dest: /etc/ssh/sshd_config.d/60-motd.conf
     content: |
        PrintMotd yes
   notify: Reload sshd to make motd changes effective
+  when: motd_show_ssh == true
```
Comment (berendt) [inline roles/motd/tasks/configure-Debian-family.yml:47]: Use `when: motd_show_ssh | bool`

File: roles/motd/tasks/configure-Debian-family.yml [hunk 2]
```diff
@@ -38,9 +38,19 @@
...
+
+- name: Configure SSH to not print the motd
+  become: true
+  file:
+    path: /etc/ssh/sshd_config.d/60-motd.conf
+    state: absent
+  notify: Reload sshd to make motd changes effective
+  when: motd_show_ssh != true
```
Comment (berendt) [inline roles/motd/tasks/configure-Debian-family.yml:55]: Use `when: not motd_show_ssh | bool`

File: roles/motd/tasks/configure-Debian-family.yml (hunk 1 above)
Comment (scoopex) [reply roles/motd/tasks/configure-Debian-family.yml]: done

**Comment by scoopex** [comment]
Hmm, according to the logs, CI complains:
(ansible-molecule: Run molecule dependency action with scenario delegated)
```
2024-05-31 09:03:50.452738 | debian-bookworm | MODULE FAILURE
failure
```
How can i get more detailed information?

File: roles/motd/tasks/configure-Debian-family.yml [hunk 3]
```diff
@@ -36,3 +36,24 @@
...
+- name: Configure SSH to print the motd
+  become: true
+  ansible.builtin.copy:
+    dest: /etc/ssh/sshd_config.d/60-motd.conf
+    content: |
+       # Shows the motd on every ssh login
+       PrintMotd yes
+    group: root
+    owner: root
+    mode: "0644"
```
Comment (berendt) [inline roles/motd/tasks/configure-Debian-family.yml:49]: use `mode: 0644`

File: roles/motd/handlers/main.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+# handlers file for motd
+- name: Reload sshd to make motd changes effective
```
Comment (berendt) [inline roles/motd/handlers/main.yml:3]: Use `Reload sshd service` as name.

File: roles/motd/handlers/main.yml [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+  become: true
+  ansible.builtin.service:
+    name: "ssh"
```
Comment (berendt) [inline roles/motd/handlers/main.yml:6]: Use `name: ssd`

File: roles/motd/tasks/configure-Debian-family.yml [hunk 6]
```diff
@@ -36,3 +36,24 @@
...
+  notify: Reload sshd to make motd changes effective
+  when: motd_show_ssh | bool
+
+- name: Configure SSH to not print the motd
+  become: true
+  ansible.builtin.file:
+    path: /etc/ssh/sshd_config.d/60-motd.conf
+    state: absent
+  notify: Reload sshd to make motd changes effective
```
Comment (berendt) [inline roles/motd/tasks/configure-Debian-family.yml:58]: Use `Reload ssh service`

File: roles/motd/tasks/configure-Debian-family.yml [hunk 7]
```diff
@@ -36,3 +36,24 @@
...
+  become: true
+  ansible.builtin.copy:
+    dest: /etc/ssh/sshd_config.d/60-motd.conf
+    content: |
+       # Shows the motd on every ssh login
+       PrintMotd yes
+    group: root
+    owner: root
+    mode: "0644"
+  notify: Reload sshd to make motd changes effective
```
Comment (berendt) [inline roles/motd/tasks/configure-Debian-family.yml:50]: Use `Reload ssh service`

File: roles/motd/handlers/main.yml (hunk 4 above)
Comment (scoopex) [reply roles/motd/handlers/main.yml]: done

File: roles/motd/tasks/configure-Debian-family.yml (hunk 3 above)
Comment (scoopex) [reply roles/motd/tasks/configure-Debian-family.yml]: fixed

**Pull request merged**